<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicWindow</class>
 <widget class="QMainWindow" name="BasicWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>BasicWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConnectHelicopter1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Helicopter 1 
Connect</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConnectHelicopter2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Helicopter 2
Connect</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>291</width>
      <height>321</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>271</width>
       <height>311</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Image Processing</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QLineEdit" name="editIP1">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>192.168.1.100</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="editIP2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>192.168.1.101</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="editIP3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>192.168.1.102</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="editIP4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>127.0.0.1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>35</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>PC1 IP:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>65</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>PC2 IP:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>95</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>PC3 IP:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>125</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>PC4 IP:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblEKFRate">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>255</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>---</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblCorrectionRate">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>230</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>---</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="editHeli1">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>192.168.1.104</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>155</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>HELI1 IP:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Measurement update:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>255</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>State update:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>211</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Time since last correction (ms):</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblTimeSinceLastCorrection">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>280</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>---</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="editHeli2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>180</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>192.168.1.110</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>185</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>HELI2 IP:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>961</width>
      <height>961</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>941</width>
       <height>941</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Data Plot</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QCustomPlot" name="qplotGraph1" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>455</width>
        <height>345</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="qplotGraph2" native="true">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>30</y>
        <width>455</width>
        <height>345</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="qplotGraph3" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>375</y>
        <width>455</width>
        <height>345</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="qplotGraph4" native="true">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>375</y>
        <width>455</width>
        <height>345</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>frame_3</zorder>
   <zorder>frame</zorder>
   <zorder>frame_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
